style/javascript/README.md

```diff
@@ -0,0 +1,18 @@
+JavaScript
+==========
+
+[Sample](sample.js)
+
+* Prefer ES6 classes over library provided classes.
```

Comment:
`React.createClass` (unless you need mixins), `_.extend`, `Backbone.Router.extend`, etc.